Jint/Runtime/Modules/ModuleLoader.cs

```diff
@@ -0,0 +1,75 @@
+﻿#nullable enable
+
+using System;
+using System.IO;
+using Esprima;
+using Esprima.Ast;
+
+namespace Jint.Runtime.Modules;
+
+public class ModuleLoader : IModuleLoader
+{
+    private readonly Uri _defaultReferencingLocation;
+
+    public ModuleLoader(string defaultBasePath)
```

Comment:
`basePath`, because there is no fallback, so "default" doesn't make much sense to me.